### ISSUE TITLE
Problem with installation w/ Laravel 5 (not dev mode)

### ISSUE THREAD

**Issue by mikeerickson** [BenConstable/phpspec-laravel#27]
Hey @BenConstable 


Running into error with installation with Laravel 5 release code.  I removed the "dev" stability setting because it caused PHPSpec to break all together.

So, it will not install without dev setting, but trying without dev causes error?   Any updates to code?

Using the following

<pre>

	"require-dev": {
          ...
        "benconstable/phpspec-laravel": "~2.0"
      },

</pre>

I receive the following error

<pre>

± % composer update                                                                                                                                                                                                                                                          !10488
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package benconstable/phpspec-laravel could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

</pre>

**Comment by BenConstable** [comment]
Hi @mikeerickson,

I've updated the phpspec dependency to reflect the stable `~2.1` release. Try the following in your `composer.json` to get the extension to install successfully:

```json
{
    "require-dev": {
        "benconstable/phpspec-laravel": "~2.0@dev"
    }
}
```

You shouldn't need the dev stability setting any longer. I've updated the [readme](https://github.com/BenConstable/phpspec-laravel/blob/master/README.md#installation) to reflect the change.

Let me know if it's working ok for you and I'll close this.
